### ISSUE TITLE
document or windows not available after import js

### ISSUE THREAD

**Issue by saurk** [gradio-app/gradio#9408]
### Describe the bug

I need to access the "js.document" but it doesn't load and says no such module

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```python
import gradio as gr
import js
from js import document

```

### System Info

```shell
chrome browser
```

### Severity

Blocking usage of gradio

**Comment by pngwn** [comment]
What is `js`?

**Comment by saurk** [comment]
it's pyodide  module.  https://pyodide.org/en/stable/usage/api/python-api.html

**Comment by pngwn** [comment]
We will need more information than this.

- Are you suing gradio lite?
- What version are you using?
- Can you create a full reproduction as the issue requests?

Closing for now, but please reopen when you can provide more details.

**Comment by saurk** [comment]
yes, I am using gradio-lite as below. I need access to baseURL like _**window.location.href**_  or **_document.baseURL_** so that I can use relative path. But f_**rom js import document, window**_ doesn't work

<html>
	<head>
		<script type="module" crossorigin src="https://cdn.jsdelivr.net/npm/@gradio/lite/dist/lite.js"></script>
		<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@gradio/lite/dist/lite.css" />
	</head>
	<body>
		<gradio-lite>
		import gradio as gr
                from js import document, window

		def greet(name):
			return "Hello, " + name + "!"

		gr.Interface(greet, "textbox", "textbox").launch()
		</gradio-lite>
	</body>
</html>

**Comment by saurk** [comment]
Let me know if I need to open a new Issue for this

**Comment by freddyaboulton** [comment]
Hi @saurk! We have stopped development of lite. You can find the last snapshot of the code here: https://github.com/gradio-app/gradio-lite . Please fork and contribute a fix 🙏
